### PR TITLE
Add md to pr-test diff

### PR DESCRIPTION
**Pull request by Gregoor**
Hey @peterbe,
I have to admit that I did not try to run this locally (most of all, because I did not know how). My intention here is to make preview work for getting #5193 into shape. Lmkwyt!

### CODE REVIEW

**Review by peterbe — COMMENTED** [review]
I always get nervous about that. 
`get-diff-action` uses [`minimatch`](https://github.com/isaacs/minimatch#minimatch) and in their example they have a `+` before the list of file extensions. 

As mentioned in Slack, I think you should include a silly change to `file/en-us/mdn/kitchensink/index.html` and make sure that file gets built.

**Comment by Gregoor** [comment]
Looks like I am pinging people to review left-and-right automatically, because the system thinks I am a stranger. Sorry @Rumyra!

File: .github/workflows/pr-test.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ jobs:
       - uses: technote-space/get-diff-action@v4.1.1
         id: git_diff_content
         with:
-          PATTERNS: files/**/*.html
+          PATTERNS: files/**/*.+(html|md)
```
Comment (peterbe) [inline .github/workflows/pr-test.yml:61]: Yay! There are so many ways and I can never remember which is which. Apparently, this here works.

File: files/en-us/mdn/kitchensink/index.html [hunk 2]
```diff
@@ -7,6 +7,8 @@
 ---
 <div>{{MDNSidebar}}</div>
 
+<h2>How about another change?</h2>
+
```
Comment (peterbe) [inline files/en-us/mdn/kitchensink/index.html:11]:
```suggestion
```

does this work?

**Review by peterbe — APPROVED** [review]
Would you mind doing another commit + push where you remove the sample change in the kitchensink page?

**Comment by Gregoor** [comment]
Done! Thanks for the hint @peterbe. Shall we merge?

**Comment by Rumyra** [comment]
> Looks like I am pinging people to review left-and-right automatically, because the system thinks I am a stranger. Sorry @Rumyra!

Ha - no worries @Gregoor as long as nothing is needed from me 👍

**Review by peterbe — APPROVED** [review]

**Pull request merged**